### ISSUE TITLE
[news] Make articles use skeleton while loading

### ISSUE THREAD

**Issue by noahmay** [Dsek-LTH/member-page#111]
Material UI has nice-looking skeletons (https://material-ui.com/components/skeleton/). This should be used to make the page look better while fetching articles.

Do this issue depend on another issue? Write Depends on #Issue_number here

[x] Relates to frontend
[ ] Relates to the backend
[ ] Requires modification of GraphQL
[ ] Requires modification of Database
[ ] Requires creating a new service
